managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/EntryImpl.java

```diff
@@ -66,6 +67,17 @@ public static EntryImpl create(long ledgerId, long entryId, byte[] data) {
         return entry;
     }
 
+    public static EntryImpl create(long ledgerId, long entryId, boolean skipped) {
+        EntryImpl entry = RECYCLER.get();
+        entry.timestamp = System.nanoTime();
+        entry.ledgerId = ledgerId;
+        entry.entryId = entryId;
+        entry.skipped = skipped;
+        entry.data = Unpooled.wrappedBuffer(new byte[0]);
```

Comment:
do we really need to allocate this ? what about using a constant ?

Reply:
entry.data = null?

Reply:
Fixed! PTAL,thanks! @eolivelli